### PR TITLE
Papers from master thesis and FRIA proposal Ilyes Gharbi

### PR DESCRIPTION
**Pull request by ilgha**

### CODE REVIEW

File: bibliography.bib [hunk 1]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    volume       = {116},
+    number       = {32},
+    year         = {2019},
+    pages        = {15849--15854},
+    doi          = {10.1073/pnas.1903070116},
+    note         = {},
+    annote       = {},
+}
+
+@Book {HasLigFraBir2019book,
```
Comment (KenN7) [inline bibliography.bib:7332]: this is duplicated
https://github.com/demiurge-project/demiurge-bib/blob/5c51777aae7c3ef2292759a85850f7395276e218/bibliography.bib#L260

File: bibliography.bib [hunk 2]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    year         = {2004},
+    pages        = {1},
+    publisher    = ACM,
+    address      = {NewYork_NY_USA},
+    doi          = {10.1145/1015330.1015430},
+    note         = {},
+    annote       = {},
+}
+
+@Article{HeaDumOsuPlaSch1998IEEEISA,
```
Comment (KenN7) [inline bibliography.bib:7163]: duplicated see below

File: bibliography.bib [hunk 3]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    year        = {2004},
+    pages       = {1},
+    publisher   = ACM,
+    address     = {NewYork_NY_USA},
+    doi         = {10.1145/1015330.1015430},
+    note        = {},
+    annote      = {},
+}
+
+@ARTICLE{HeaDumOsu-etal1998IEEEISA,
```
Comment (KenN7) [inline bibliography.bib:7376]: This is duplicated at line 7163

File: bibliography.bib [hunk 4]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    volume       = {13},
+    number       = {4},
+    year         = {1998},
+    pages        = {18--28},
+    doi          = {10.1109/5254.708428},
+    note         = {},
+    annote       = {},
+}
+
+@inproceedings{AkrSchSeb2012ecmlpkdd,
```
Comment (KenN7) [inline bibliography.bib:7390]: duplicated line 7177

File: bibliography.bib [hunk 5]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    number       = {},
+    year         = {2012},
+    pages        = {116--131},
+    doi          = {10.1007/978-3-642-33486-3_8},
+    note         = {},
+    annote       = {},
+}
+
+@inproceedings{AkrSchSeb2011ecmlpkdd,
+    title        = {Preference-based policy learning},
```
Comment (KenN7) [inline bibliography.bib:7405]: duplicated line 7192

File: bibliography.bib [hunk 6]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    number       = {},
+    year         = {2011},
+    pages        = {12--27},
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@inproceedings{NgRus2000icml,
+    title        = {Algorithms for inverse reinforcement learning},
```
Comment (KenN7) [inline bibliography.bib:7419]: duplicated line 7206

and a lot of the following also

File: bibliography.bib [hunk 7]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    year         = {2006},
+    pages        = {363--372},
+    publisher    = pub_SP,
+    address      = {},
+    doi          = {10.1007/11552246_35},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {busoniu2006decentralized,
```
Comment (KenN7) [inline bibliography.bib:7575]: the key is bad

File: bibliography.bib [hunk 8]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    address      = {},
+    doi          = {10.1109/ICARCV.2006.345351},
+    note         = {},
+    annote       = {},
+}
+
+@Article{KobBagPet2011ijrr,
+    title        = {Reinforcement learning in robotics: a survey},
+    author       = Kober_J # and # Bagnell_JA # and # Peters_J,
+    journal      = IJRR,
```
Comment (KenN7) [inline bibliography.bib:7595]: journals the identifier should be capitalized (e.g. SI),

File: bibliography.bib [hunk 9]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    year         = {2020},
+    pages        = {933--937},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/SII46433.2020.9025873},
+    note         = {},
+    annote       = {},
+}
+
+@Article{OsaPajNeu-etal2018ftr,
```
Comment (KenN7) [inline bibliography.bib:7688]: journals the identifier should be capitalized (e.g. SI),

File: bibliography.bib [hunk 10]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    pages        = {},
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@Article{SosKhuZou-etal2016arxiv,
+    title        = {Inverse reinforcement learning in swarm systems},
+    author       = Sosic_A # and # KhudaBukhsh_WR # and # Zoubir_AM # and # Koeppl_H,
+    journal      = {arXiv preprint arXiv:1602.05450},
```
Comment (KenN7) [inline bibliography.bib:7750]: remove doi

File: bibliography.bib [hunk 11]
```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
...
+    volume       = {},
+    number       = {},
+    year         = {2016},
+    pages        = {},
+    doi          = {10.48550/arXiv.1602.05450},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BahGupPat2022arxiv,
```
Comment (KenN7) [inline bibliography.bib:7761]: see https://github.com/demiurge-project/demiurge-bib/blob/5c51777aae7c3ef2292759a85850f7395276e218/bibliography.bib#L2822

for arxiv papers

File: journal.bib [hunk 12]
```diff
@@ -31,11 +34,18 @@ @string{ COMP
...
 @string{ EREI        = "Economic Research-Ekonomska Istraživanja" }
 @string{ EJOR        = "European Journal of Operational Research" }
 @string{ ETT         = "European Transactions on Telecommunications" }
 @string{ EC          = "Evolutionary Computation" }
 @string{ EI          = "Evolutionary Intelligence" }
+@string{ ERIX        = "Experimental robotics IX" }
+@string{ ETT         = "European Transactions on Telecommunications" }
+@string{ FRAI        = "Frontiers in Robotics and AI" }
+@string{ FTR         = "Foundations and Trends{\textregistered} in Robotics" }
+@string{ HX          = "HardwareX" }
```
Comment (KenN7) [inline journal.bib:47]: already present 3 lines below...

File: journal.bib [hunk 13]
```diff
@@ -31,11 +34,18 @@ @string{ COMP
...
 @string{ CB          = "Current Biology" }
+@string{ CSUR        = "ACM Computing Surveys (CSUR)" }
 @string{ EREI        = "Economic Research-Ekonomska Istraživanja" }
 @string{ EJOR        = "European Journal of Operational Research" }
 @string{ ETT         = "European Transactions on Telecommunications" }
 @string{ EC          = "Evolutionary Computation" }
 @string{ EI          = "Evolutionary Intelligence" }
+@string{ ERIX        = "Experimental robotics IX" }
+@string{ ETT         = "European Transactions on Telecommunications" }
+@string{ FRAI        = "Frontiers in Robotics and AI" }
```
Comment (KenN7) [inline journal.bib:45]: duplicated 3 lines below

**Comment by KenN7** [comment]
Most of the keys for journals are wrong, also check the bib commands like "@InProceedings" most of them are not capitalised.
Half of the papers  appear twice

**Comment by ilgha** [comment]
Thank you @KenN7 for your comments!

**Comment by KenN7** [comment]
Another advice: idk how familiar you are with git, but it would be great if the commit history was clean.
I mean, having commits that add and then remove the same things only add confusion to the repo's history.
you should lookup "git rebase" in order to rewrite your history and only leave one commit there.
If you rebase the history on your repo, your PR should normally be updated .

File: author.bib [hunk 14]
```diff
@@ -18,16 +18,19 @@ @comment{As
 @string{ Aarts_E              = "Aarts, Emile" }
 @string{ Abate_M              = "Abate, Marcus" }
 @string{ Abdullah_S           = "Abdullah, Salwani" }
+@string{ Abbeel_P             = "Abbeel, Pieter" }
```
Comment (glegarda) [inline author.bib:21]: Should appear before Abdullah_S

File: journal.bib [hunk 15]
```diff
@@ -52,6 +59,7 @@ @string{ IEEETRA
 @string{ IEEETSMC    = "IEEE Transactions on Systems, Man, and Cybernetics" }
 @string{ IEEETSMCB   = "IEEE Transactions on Systems, Man, and Cybernetics, Part B (Cybernetics)" }
 @string{ IPV         = "IFAC Proceedings Volumes" }
+@string{ IMASA       = "Innovations in multi-agent systems and applications-1" }
```
Comment (glegarda) [inline journal.bib:62]:
```suggestion
@string{ IMASA       = "Innovations in Multi-Agent Systems and Applications -- 1" }
```

File: journal.bib [hunk 16]
```diff
@@ -66,20 +74,25 @@ @string{ JPAMT
 @string{ JSA         = "Journal of Systems Architecture" }
 @string{ JASA        = "Journal of the American Statistical Association" }
 @string{ MACH        = "Machines" }
+@string{ ML          = "Machine Learning" }
+@string{ MCMDS       = "Mathematical and Computer Modelling of Dynamical Systems" }
+@string{ MICPRO      = "Microprocessors and Microsystems - Embedded Hardware Design" }
```
Comment (glegarda) [inline journal.bib:79]: Duplicate (see 5 lines below)

File: journal.bib [hunk 17]
```diff
@@ -66,20 +74,25 @@ @string{ JPAMT
...
 @string{ MCMDS       = "Mathematical and Computer Modelling of Dynamical Systems" }
 @string{ MP          = "Mathematical Programming" }
 @string{ MOOR        = "Mathematics of Operations Research" }
 @string{ MC          = "Memetic Computing" }
 @string{ MICPRO      = "Microprocessors and Microsystems - Embedded Hardware Design" }
 @string{ NC          = "Natural Computing" } 
 @string{ NATU        = "Nature" }
+@string{ NATUBIOT    = "Nature biotechnology" }
```
Comment (glegarda) [inline journal.bib:87]:
```suggestion
@string{ NATUBIOT    = "Nature Biotechnology" }
```

File: journal.bib [hunk 18]
```diff
@@ -66,20 +74,25 @@ @string{ JPAMT
 @string{ JSA         = "Journal of Systems Architecture" }
 @string{ JASA        = "Journal of the American Statistical Association" }
 @string{ MACH        = "Machines" }
+@string{ ML          = "Machine Learning" }
```
Comment (glegarda) [inline journal.bib:77]: Should go before MACH

File: journal.bib [hunk 19]
```diff
@@ -66,20 +74,25 @@ @string{ JPAMT
...
 @string{ NATU        = "Nature" }
+@string{ NATUBIOT    = "Nature biotechnology" }
 @string{ NATUCOM     = "Nature Communications" }
 @string{ NATUMINT    = "Nature Machine Intelligence" }
 @string{ NET         = "Networks" }
 @string{ NCO         = "Neural Computation" }
 @string{ NN          = "Neural Networks" }
 @string{ ORE         = "Open Research Europe" }
 @string{ ORP         = "Operations Research Perspectives" }
+@string{ PACMPL      = "Proceedings of the ACM on Programming Languages" }
```
Comment (glegarda) [inline journal.bib:95]: Duplicate (see 6 lines below)

File: journal.bib [hunk 20]
```diff
@@ -93,6 +106,7 @@ @string{ RAIRO
 @string{ RMP         = "Reviews of Modern Physics" }
 @string{ ROB         = "Robotica" }
 @string{ RAS         = "Robotics and Autonomous Systems" }
+@string{ RMP         = "Reviews of Modern Physics" }
```
Comment (glegarda) [inline journal.bib:109]: Duplicate (see 3 lines above)

File: publisher.bib [hunk 21]
```diff
@@ -16,6 +16,7 @@ @string{ pub_BB
 @string{ pub_CEURWS    = "CEUR Workshop Proceedings" }
 @string{ pub_CRCP      = "CRC Press" }
 @string{ pub_CSREA     = "CSREA Press" } @comment{Computer Science Research, Education, and Applications Press (CSREA)}
+@string{ pub_El        = "Elsevier" }
```
Comment (glegarda) [inline publisher.bib:19]:
```suggestion
@string{ pub_EL        = "Elsevier" }
```

File: proceedings.bib [hunk 22]
```diff
@@ -12,6 +12,7 @@ @comment{Here
 @comment{é ß ó ñ á ü Á ë ú ó Ş ö ä ã ç č ø ł è ô}
 
 @string{ aaai_2006      = "AAAI'06: Proceedings of the 21st national conference on Artificial intelligence" }
+@string{ aaai_2021      = "AAAI'21: Proceedings of the 36st national conference on Artificial intelligence" }
```
Comment (glegarda) [inline proceedings.bib:15]:
```suggestion
@string{ aaai_2021      = "AAAI'21: Proceedings of the 36th national conference on Artificial intelligence" }
```

File: proceedings.bib [hunk 23]
```diff
@@ -48,6 +49,7 @@ @string{ cec_2005
 @string{ cec_2009       = "2009 IEEE Congress on Evolutionary Computation" }
 @string{ cec_2015       = "2015 IEEE Congress on Evolutionary Computation (CEC)" }
 @string{ cec_2018       = "2018 IEEE Congress on Evolutionary Computation (CEC)" }
+@string{ corl_2017      = "2017 Conference on robot learning" }
```
Comment (glegarda) [inline proceedings.bib:52]:
```suggestion
@string{ corl_2017      = "2017 Conference on Robot Learning" }
```

File: proceedings.bib [hunk 24]
```diff
@@ -62,6 +64,8 @@ @string{ ecal_1995
 @string{ ecal_2001      = "Advances in Artificial Life: 6th European Conference, ECAL 2001" }
 @string{ ecal_2003      = "Advances in Artificial Life: 7th European Conference, ECAL 2003" }
 @string{ ecal_2017      = "ECAL 2017, the Fourteenth European Conference on Artificial Life " }
+@string{ ecmlpkdd_2011  = "2011 Joint European conference on machine learning and knowledge discovery in databases" }
```
Comment (glegarda) [inline proceedings.bib:67]:
```suggestion
@string{ ecmlpkdd_2011  = "2011 Joint European Conference on Machine Learning and Knowledge Discovery in Databases" }
```

File: proceedings.bib [hunk 25]
```diff
@@ -62,6 +64,8 @@ @string{ ecal_1995
...
+@string{ ecmlpkdd_2012  = "2012 Joint European conference on machine learning and knowledge discovery in databases" }
```
Comment (glegarda) [inline proceedings.bib:68]:
```suggestion
@string{ ecmlpkdd_2012  = "2012 Joint European Conference on Machine Learning and Knowledge Discovery in Databases" }
```

File: proceedings.bib [hunk 26]
```diff
@@ -87,11 +91,16 @@ @string{ ica_2016
...
+@string{ icarcv_2006    = "2006 9th International Conference on Control Automation Robotics \& Vision (ICARCV)" }
 @string{ icarcv_2012    = "2012 12th International Conference on Control Automation Robotics \& Vision (ICARCV)" }
 @string{ iccad_1995     = "ICCAD'95: Proceedings of the 1995 IEEE/ACM international conference on Computer-aided design" }
+@string{ iccv_2019      = "2019 Proceedings of the IEEE/CVF International Conference on Computer Vision" }
 @string{ iciis_1999     = "1999 International Conference on Information Intelligence and Systems" }
 @string{ iciis_2011     = "2011 6th International Conference on Industrial and Information Systems, ICIIS 2011" }
 @string{ icinco_2011    = "Informatics in Control Automation and Robotics: Revised and Selected Papers from the International Conference on Informatics in Control Automation and Robotics 2009" }
+@string{ icml_2000      = "ICML '00: Proceedings of the Seventeenth International Conference on Machine Learning" }
+@string{ icml_2004      = "ICML '04: Proceedings of the twenty-first international conference on Machine learning" }
```
Comment (glegarda) [inline proceedings.bib:102]:
```suggestion
@string{ icml_2004      = "ICML '04: Proceedings of the Twenty-first International Conference on Machine Learning" }
```

File: proceedings.bib [hunk 27]
```diff
@@ -102,7 +111,9 @@ @string{ icra_2015
 @string{ icra_2017      = "2017 IEEE International Conference on Robotics and Automation (ICRA)" }
 @string{ icra_2020      = "2020 IEEE International Conference on Robotics and Automation (ICRA)" }
 @string{ icra_2021      = "2021 IEEE International Conference on Robotics and Automation (ICRA)" }
+@string{ icra_2022      = "2022 IEEE International Conference on Robotics and Automation (ICRA)" }
 @string{ ijcai_2019     = "Proceedings of the Twenty-Eighth International Joint Conference on Artificial Intelligence, IJCAI-19" }
+@string{ irc_2018       = "2018 second IEEE international conference on robotic computing (IRC)" }
```
Comment (glegarda) [inline proceedings.bib:116]:
```suggestion
@string{ irc_2018       = "2018 second IEEE International Conference on Robotic Computing (IRC)" }
```

**Review by glegarda — COMMENTED** [review]
There are quite a few errors, including duplicate entries, improper capitalisation and ordering, and even mismatches between author names and their corresponding keys. Before this PR is accepted (and also for the next time), please go over all the changes carefully.

**Comment by glegarda** [comment]
Also, please add the corresponding entries to all "-short.bib" files.

File: author.bib [hunk 28]
```diff
@@ -453,6 +452,7 @@ @string{ Hornby_GS
 @string{ Horner_J             = "Hörner, Jiří" }
 @string{ Hornik_K             = "Hornik, Kurt" }
 @string{ Howard_A             = "Howard, Andrew" }
+@string{ Hsu_D                = "Hsu, Daniel" }
```
Comment (ilgha) [inline author.bib:455]: It is still in the wrong place

File: author.bib [hunk 29]
```diff
@@ -600,7 +598,7 @@ @string{ Liu_TY
 @string{ Liu_W                = "Liu, Wenguo" }
 @string{ Lochmatter_T         = "Lochmatter, Thomas" }
 @string{ Longchamp_V          = "Longchamp, Valentin" }
-@string{ Lopes_AM              = "Lopez, Antonio M" }
+@string{ Lopes_AM              = "Lopez, Antonio M." }
```
Comment (ilgha) [inline author.bib:601]: Lopez_AM*

File: bibliography.bib [hunk 30]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Natarajan_S # and # Kunapuli_G # and # Judah_K # and # Tadepalli_P # and # Kersting_K # and # Shavlik_J,
+    booktitle    = icmla_2010,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2010},
+    pages        = {395--400},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7514]: Missing address

File: bibliography.bib [hunk 31]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Finn_C # and # Yu_T # and # Zhang_T # and # Abbeel_P # and # Levine_S,
+    booktitle    = corl_2017,
+    editor       = Levine_S # and # Vanhoucke_V # and # Goldberg_K,
+    series       = {},
+    volume       = {78},
+    number       = {},
+    year         = {2017},
+    pages        = {357--368},
+    publisher    = {},
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7577]: Missing publisher and address

File: bibliography.bib [hunk 32]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Codevilla_F # and # Santana_E # and # Lopez_AM # and # Gaidon_A,
+    booktitle    = iccv_2019,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2019},
+    pages        = {9329--9338},
+    publisher    = {},
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7622]: Missing publisher and address

File: bibliography.bib [hunk 33]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Ng_AY # and # Coates_A # and # Diel_M # and # Ganapathi_V # and # Schulte_J # and # Tse_B # and # Berger_E # and # Liang_E,
+    booktitle    = ERIX,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2006},
+    pages        = {363--372},
+    publisher    = pub_SP,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7370]: Missing address

File: bibliography.bib [hunk 34]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Busoniu_L # and # DeSchutter_B # and # Babuska_R,
+    booktitle    = icarcv_2006,
+    editor       = Assmann_U # and # Wagner_G,
+    series       = {},
+    volume       = {1319},
+    number       = {},
+    year         = {2006},
+    pages        = {1--6},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7387]: Missing address

File: bibliography.bib [hunk 35]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Yasuda_T # and # Ohkura_K,
+    booktitle    = irc_2018,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2018},
+    pages        = {179--180},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7418]: Missing address

File: bibliography.bib [hunk 36]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Zhao_Z # and # Wang_Z # and # Han_K # and # Gupta_R # and # Tiwari_P # and # Wu_G # and # Barth_MJ,
+    booktitle    = icra_2022,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2022},
+    pages        = {2891--2897},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7449]: Missing address

File: bibliography.bib [hunk 37]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Zhou_L # and # Small_K,
+    booktitle    = aaai_2021,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2021},
+    pages        = {11116--11124},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7466]: Missing address

File: bibliography.bib [hunk 38]
```diff
@@ -7142,3 +7142,485 @@ @Article {ProHsiKum2017IEEETR
...
+    author       = Hansen_EB # and # Andersen_RE # and # Madsen_S # and # Bogh_S,
+    booktitle    = sii_2020,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2020},
+    pages        = {933--937},
+    publisher    = pub_IEEE,
+    address      = {},
```
Comment (jonaskuResearch) [inline bibliography.bib:7483]: Missing address

**Review by jonaskuResearch — CHANGES_REQUESTED** [review]
Please check publisher and address for all added entries in the bibliography.

**Pull request closed without merging**